### PR TITLE
Add functionality to read from https and support reduced files.

### PR DESCRIPTION
**Pull request by pwrose**
This resolved issue #34

### CODE REVIEW

**Comment by pwrose** [comment]
This also resolved #33

File: mmtf-serialization/src/main/java/org/rcsb/mmtf/utils/CodecUtils.java [hunk 1]
```diff
@@ -14,14 +14,55 @@
 	/** The base url for this version of MMTF. */
 	public static final String BASE_URL = "http://mmtf.rcsb.org/v1.0/full/";
```
Comment (josemduarte) [inline mmtf-serialization/src/main/java/org/rcsb/mmtf/utils/CodecUtils.java]: This constant is not needed anymore, is it?

**Review by josemduarte — CHANGES_REQUESTED** [review]
Just one minor modification, looks great otherwise!

**Comment by pwrose** [comment]
Removed the unused constant.

**Review by josemduarte — APPROVED** [review]

**Pull request merged**